sros2/sros2/verb/generate_artifacts.py

```diff
@@ -44,8 +48,43 @@ def add_arguments(self, parser, cli_name):
 
     def main(self, *, args):
         try:
-            success = generate_artifacts(
+            success = _generate_artifacts(
```

Comment:
I think of the content of the "verb" files as not public API. Like it's used by that verb but it it not in the API module.
It would be convenient for function like generate_artifacts to be in the api module to be usable by code outside sros2.
Right now multiple pieces of code (`sros2_cmake` and `test_security`) call the CLI, but if any was in python they could use this API instead

Reply:
Alright I'll rework this to be more similar to the others.

Reply:
Done.